Loop/Views/BolusEntryView.swift

```diff
@@ -293,13 +332,13 @@ struct BolusEntryView: View, HorizontalSizeClassOverride {
 
     private var actionArea: some View {
         VStack(spacing: 0) {
-            if isNoticeVisible {
+            if isNoticeVisible && !viewModel.isLoggingDose {
                 warning(for: viewModel.activeNotice!)
                     .padding([.top, .horizontal])
                     .transition(AnyTransition.opacity.combined(with: .move(edge: .bottom)))
             }
 
-            if isManualGlucosePromptVisible {
+            if isManualGlucosePromptVisible && !viewModel.isLoggingDose {
```

Comment:
I made it so logging a dose doesn't prompt for a manual BG, but I don't feel strongly about this